[2025-05-28T12:17:31.113+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:55:00+00:00 [queued]>
[2025-05-28T12:17:31.137+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:55:00+00:00 [queued]>
[2025-05-28T12:17:31.138+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-28T12:17:31.160+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): transform_weather_task> on 2025-05-28 10:55:00+00:00
[2025-05-28T12:17:31.167+0000] {standard_task_runner.py:60} INFO - Started process 1734 to run task
[2025-05-28T12:17:31.173+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_data_etl', 'transform_weather_task', 'scheduled__2025-05-28T10:55:00+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmp7v1yvzk9']
[2025-05-28T12:17:31.180+0000] {standard_task_runner.py:88} INFO - Job 60: Subtask transform_weather_task
[2025-05-28T12:17:31.239+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_data_etl.transform_weather_task scheduled__2025-05-28T10:55:00+00:00 [running]> on host f1bd559e8a8b
[2025-05-28T12:17:31.318+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data_etl' AIRFLOW_CTX_TASK_ID='transform_weather_task' AIRFLOW_CTX_EXECUTION_DATE='2025-05-28T10:55:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-28T10:55:00+00:00'
[2025-05-28T12:17:31.511+0000] {python.py:201} INFO - Done. Returned value was: Data cuaca berhasil ditransformasi dengan 5 baris data
[2025-05-28T12:17:31.527+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_data_etl, task_id=transform_weather_task, execution_date=20250528T105500, start_date=20250528T121731, end_date=20250528T121731
[2025-05-28T12:17:31.546+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-28T12:17:31.562+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
